samples/cpp/imagelist_creator.cpp

```diff
@@ -1,54 +0,0 @@
-/*this creates a yaml or xml list of files from the command line args
```

Comment:
I propose to presume it. Image lists are used here and there as input for other tools and samples.